config/crd/release.giantswarm.io_releases.yaml

```diff
@@ -164,9 +169,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
-  acceptedNames:
-    kind: ""
-    plural: ""
-  conditions: []
-  storedVersions: []
```

Comment:
Why is this removed?

Reply:
this was done by `controller-gen` so I don't know why that's the case.

Reply:
hmm ok, let's merge it 👍🏻